### ISSUE TITLE
Stopped updating?

### ISSUE THREAD

**Issue by fortinmike** [matthewpalmer/Charter#46]
It seems the app doesn't display any message that was sent after June 8th.

I tried uninstalling the reinstalling the app but the issue didn't go away. Maybe there's something wrong with the app's backend?

**Comment by matthewpalmer** [comment]
Yeah there's something going on in the backend after a recent update, haven't been able to work it out yet. Thanks for letting me know, I'll try to follow it up soon

**Comment by matthewpalmer** [comment]
Should be back up now, but who knows how long it'll last :/. There'll be a gap of like 8 days of no updates where we missed the emails—the email listener says it put them in the database without error but for the life of me I can't figure out where they went. I'll leave this open for a few days, and keep my fingers crossed.

Feel free to comment again if it goes down—I don't check Charter every day.

**Comment by fortinmike** [comment]
FYI, this happened again. It's been at least a week or two since Charter has worked.
